src/lib/components/admin/Settings/Users.svelte

```diff
@@ -29,8 +29,8 @@
 
```

Comment:
I'm guessing this line means I don't need to change anything in the storage layer to ensure the whitelist setting is used for the allowlist now.